SPECS/libvirt/libvirt.spec

```diff
@@ -1,976 +1,113 @@
-# Force QEMU to run as non-root
-%define qemu_user  qemu
-%define qemu_group  qemu
-
-%bcond_with missing_dependencies
-
 Summary:        Virtualization API library that supports KVM, QEMU, Xen, ESX etc
 Name:           libvirt
 Version:        6.1.0
-Release:        3%{?dist}
+Release:        2%{?dist}
```

Comment:
We didn't have a successful build, so reverting the `Release` tag is fine as well.